samples/token-server/index.js

```diff
@@ -0,0 +1,108 @@
+require('dotenv').config();
+
+const { randomBytes } = require('crypto');
+const fetch = require('node-fetch');
+const restify = require('restify');
+
+// Create server
+const server = restify.createServer();
+
+server.listen(process.env.PORT, () => {
+  console.log(`${ server.name } listening to ${ server.url }`);
+});
+
+server.use(restify.plugins.queryParser());
+
+server.get('/health.txt', async (req, res) => {
+  res.set('Content-Type', 'text/plain');
+  res.send('OK');
+});
+
+async function createUserID() {
+  return new Promise((resolve, reject) => {
+    randomBytes(16, (err, buffer) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(`dl_${ buffer.toString('hex') }`);
+      }
+    })
+  });
+}
+
+server.post('/directline/token', async (req, res) => {
+  const origin = req.header('origin');
+  const userID = await createUserID();
+  const { token } = req.query;
+
+  if (token) {
+    console.log(`Refreshing Direct Line token for ${ origin }`);
+  } else {
+    console.log(`Requesting Direct Line token for ${ origin }`);
+  }
+
+  try {
+    let cres;
+
+    if (token) {
+      cres = await fetch('https://directline.botframework.com/v3/directline/tokens/refresh', {
+        // body: JSON.stringify({ TrustedOrigins: [origin] }),
```

Comment:
Is this supposed to be commented out and not removed from the sample? If so, please add a comment above this line explaining what it's for and when a user should uncomment it.